rules/windows/process_creation/win_malware_script_dropper.yml

```diff
@@ -16,22 +16,22 @@ logsource:
     product: windows
 detection:
     selection:
-        Image:
-            - '*\wscript.exe'
-            - '*\cscript.exe'
-        CommandLine:
-            - '* C:\Users\\*.jse *'
-            - '* C:\Users\\*.vbe *'
-            - '* C:\Users\\*.js *'
-            - '* C:\Users\\*.vba *'
-            - '* C:\Users\\*.vbs *'
-            - '* C:\ProgramData\\*.jse *'
-            - '* C:\ProgramData\\*.vbe *'
-            - '* C:\ProgramData\\*.js *'
-            - '* C:\ProgramData\\*.vba *'
-            - '* C:\ProgramData\\*.vbs *'
+        Image|endswith:
+            - '\wscript.exe'
+            - '\cscript.exe'
+        CommandLine|contains:
+            - ' C:\Users\\*.jse '
+            - ' C:\Users\\*.vbe '
+            - ' C:\Users\\*.js '
+            - ' C:\Users\\*.vba '
+            - ' C:\Users\\*.vbs '
+            - ' C:\ProgramData\\*.jse '
+            - ' C:\ProgramData\\*.vbe '
+            - ' C:\ProgramData\\*.js '
+            - ' C:\ProgramData\\*.vba '
+            - ' C:\ProgramData\\*.vbs '
```

Comment:
Could you please change it to the following:

```
detection:
    selection1:
        Image|endswith:
            - '\wscript.exe'
            - '\cscript.exe'
        CommandLine|contains:
            - 'C:\Users\'
            - 'C:\ProgramData\'
    selection2:
        CommandLine|contains:
            - '.jse'
            - '.vbe'
            - '.js'
            - '.vba'
            - '.vbs'
    falsepositive:
        ParentImage|contains: '\winzip'
    condition: selection1 and selection2 and not falsepositive
```

Reply:
Applied ;)